config/base/params.env

```diff
@@ -1,6 +1,6 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.12.0
+odh-model-controller=quay.io/opendatahub/odh-model-controller:stable
 caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:stable-2d9db23
```

Comment:
Why we are commiting stable tag in main branch?
stable tag are created from release branch. Fast tags are created from main branch.

Reply:
we have changed the main branch to stable and the incubating branch will be the new branch that receives PRs.